lib/modules/manager/helm-values/types.ts

```diff
@@ -1,5 +1,7 @@
-export type HelmDockerImageDependency = {
+type HDIDBasic = {
```

Comment:
👎 Use long names, prefer interfaces and export public types

Reply:
Thanks, should be fixed now